challenge/src/main/java/e1/Exercise1Impl.java

```diff
@@ -0,0 +1,16 @@
+package e1;
+
+public class Exercise1Impl implements Exercise1 {
+
+    @Override
+    public String verify(Integer number){
+
+        if((number % 2) == 0){
+            return "Even";
+        }
+        else {
```

Comment:
O que acha de remover o else e chamar o return direto?? 
```suggestion
        return "Odd";
```